packages/node-cli/package.json

```diff
@@ -32,7 +32,7 @@
 		"chalk": "4.1.2",
 		"cli-spinner": "0.2.10",
 		"commander": "11.1.0",
-		"inquirer": "8.2.6",
+		"inquirer": "10.1.2",
```

Comment:
Is this correct? Does the interactive questions work ok with this update?

Reply:
You're right. Reverted